src/components/Button/README.md

```diff
@@ -373,6 +373,22 @@ Use when there is only one primary action but other related actions can be taken
 </div>
 ```
 
+### Button with Icon
```

Comment:
I noticed there were 2 code blocks in this example. I believe you'll want to have a single code block per example for it to be counted a story in Storybook.

Reply:
Ahh, thanks, wasn't familiar with this!